jobbergate-cli/jobbergate_cli/end_to_end_testing/applications.py

```diff
@@ -0,0 +1,86 @@
+import json
+from dataclasses import dataclass, field
+from pathlib import Path
+from typing import List
+
+import buzz
+
+from jobbergate_cli.end_to_end_testing.base import BaseEntity
+from jobbergate_cli.end_to_end_testing.constants import APPLICATIONS_CACHE_PATH, TEST_APPLICATIONS_PATH
+from jobbergate_cli.end_to_end_testing.utils import cached_run
+
+
+def get_application_list() -> List[Path]:
+    return [p for p in TEST_APPLICATIONS_PATH.iterdir() if p.is_dir()]
+
+
+def get_test_applications() -> List[Path]:
+    return list(APPLICATIONS_CACHE_PATH.glob("*.json"))
+
+
+@dataclass
+class Applications(BaseEntity):
+    base_applications: List = field(default_factory=get_application_list)
+
+    def create(self):
+        for app in self.base_applications:
+            name = app.name
+            identifier = str(Path.cwd().name) + "-" + name
```

Comment:
you can just use f-string for this:
```
indentifier = f"{Path.cwd().name}-{name}"
```

Feels a little cleaner to me.

Reply:
solved in [a990526](https://github.com/omnivector-solutions/jobbergate/pull/225/commits/a990526055f5f08f7dc732cce337cbc5fedb923e)